examples/onnxruntime/training/text-classification/README.md

```diff
@@ -39,14 +39,29 @@ python run_glue.py \
     --output_dir /tmp/ort-bert-sst2/
 ```
 
+### Performance
+
+We get the following results for [roberta-base](https://huggingface.co/roberta-base) and [roberta-large](https://huggingface.co/roberta-large) 
+on sst2 dataset under PyTorch and ONNX Runtime backends. (a single Nvidia A100 was used):
+
+| Run configuration            | FP16 | Runtime | Train samples(/s) |
+| ---------------------------- | ---- | ------- | ----------------- |
+| roberta-base / PyTorch       | TRUE | 752.3   | 268.589           |
```

Comment:
If FP16 always set to True, we can remove this column and note this information in the introduction sentence before the table. We can even create an other column to separate the informations displayed in the column "Run configuration"

Reply:
Good point!